### ISSUE TITLE
Exception: The scheme `href='https` is invalid. 

### ISSUE THREAD

**Issue by jeffbotw** [fivefilters/readability.php#14]
I'm getting a lot of errors related to the URI lib, such as: 
```Exception: The scheme `href='https` is invalid. /home/[...REMOVED...]/public_html/application/third_party/Readability.php/v3.1.2/vendor/league/uri/src/Uri.php 249```

**Comment by fivefilters** [comment]
Thanks for reporting this @jeffbotw. Are you able to provide the code you used that led to the error so we can try to reproduce it. If not, can you provide the URL of the page which caused the error?

**Comment by fivefilters** [comment]
We were able to reproduce this. It ocurred when Readability.php was set to rewrite relative URL:

```php
$configuration = new Configuration();
$configuration
    ->setFixRelativeURLs(true)
    ->setOriginalURL('http://example.com/article.html');
$readability = new Readability($configuration);
```
and the HTML being parsed contained invalid URL, e.g.
```html
<article>
    <h1>Title</h1>
    <p>Paragraph of text.</p>
    <p>Paragraph with bad link: <a href="href='https:/bad'">test</a></p>
</article>
```

Fixed in 3.1.3: https://github.com/fivefilters/readability.php/releases/tag/v3.1.3
